WooCommerce/Classes/ViewRelated/Products/Cells/ProductsTabProductTableViewCell.swift

```diff
@@ -58,6 +58,14 @@ final class ProductsTabProductTableViewCell: UITableViewCell {
         super.updateConfiguration(using: state)
         updateDefaultBackgroundConfiguration(using: state)
     }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        syncingOverlayView?.removeFromSuperview()
+        selectedProductImageOverlayView?.removeFromSuperview()
+        syncingOverlayView = nil
+        selectedProductImageOverlayView = nil
+    }
```

Comment:
Attempt an additional fix for the [reuse issue](https://github.com/woocommerce/woocommerce-ios/pull/15052#issuecomment-2649713593) mentioned in #15052.

Reply:
This seems to have fixed the issue, I'm not able to replicate it anymore! 👏